api/v1/tests.py

```diff
@@ -1,3 +1,68 @@
-from django.test import TestCase
+from django.contrib.auth import get_user_model
+from django.urls import reverse_lazy
+
+from rest_framework.test import APITestCase
+from rest_framework import status
+
+from fixtureless.factory import create
+
+from recipes.models import Recipe
+
+
+class TestRecipeApi(APITestCase):
+    def setUp(self):
+        self.user = get_user_model().objects.create(username='test_user')
+        self.recipe = create(Recipe, {'title': 'testing'})
+
+    def test_recipe_list(self):
+        """
+        Verify GET returns Recipe list
+        """
+        response = self.client.get(reverse_lazy('api:v1:recipe-list'))
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+    def test_create_account(self):
+        """
+        Verify POST creates a Recipe
+        """
+        url = reverse_lazy('api:v1:recipe-list')
+        data = {
+            "user": 1,
+            "title": "testing",
+            "description": "test description",
+            "ingredients": [],
+            "preparation_time": "00:00:01.500000",
+            "cook_time": "00:00:03",
+            "servings": 2,
+            "max_servings": 3,
+            "difficulty": 1
+        }
+        response = self.client.post(url, data, format='json')
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+
+    def test_recipe_detail(self):
+        """
+        Verify GET returns Recipe detail
+        """
+        response = self.client.get(reverse_lazy('api:v1:recipe-detail', kwargs={'pk': self.recipe.pk}))
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(response.data['title'], 'testing')
+
+    def test_recipe_update(self):
+        """
+        Verify PATCH updates our Recipe
+        """
+        response = self.client.patch(reverse_lazy('api:v1:recipe-detail',
```

Comment:
I think this still violates line length limits. By assigning the url for detail in the setup, you can shorten this. You can also indent this further toward the left.

```
self.client.patch(
    reverse_lazy('api:v1:recipe-detail', kwargs={'pk': self.recipe.pk}), {'title': 'new_test_title'})
```

Personally, this indentation style that I've seen called "Tomahawk indentation" (because it looks like the handle followed by the head of a tomahawk) is my least favorite among the pep8 allowed indentations, because it doesn't give you as much breathing room from line length limits as other styles, but also because it creates this massive leading whitespace before the indentation that is easily missed when looking over a file.